src/plugins/intel_cpu/src/.clang-tidy

```diff
@@ -113,6 +109,10 @@ WarningsAsErrors: '*'
 FormatStyle: file
 HeaderFilterRegex: 'src/plugins/intel_cpu/src/.*\.h(pp)?$'
 CheckOptions:
+  - key: bugprone-narrowing-conversions.IgnoreConversionFromTypes
+    value: "size_t;ptrdiff_t;size_type;difference_type;value_type"
```

Comment:
I propose to add uint32_t and uint64_t here as well, since this is basically the same situation as converting from size_t

Reply:
Done, added these types in several variations